Upstream only uses the version from the VERSIONFILE if git-describe(1)
fails. Unfourtunatly, git is installed on the builders and therefore the
invocation won't fail. Patch it to not invoke git at all and always use
the VERSIONFILE.

diff -urp hy-0.17.0.orig/get_version.py hy-0.17.0/get_version.py
--- hy-0.17.0.orig/get_version.py	2019-08-25 12:19:03.420044611 +0200
+++ hy-0.17.0/get_version.py	2019-08-25 12:19:12.650057246 +0200
@@ -5,16 +5,7 @@ import os, subprocess, runpy
 os.chdir(os.path.split(os.path.abspath(__file__))[0])
 VERSIONFILE = os.path.join("hy", "version.py")
 
-try:
-    __version__ = (subprocess.check_output
-                   (["git", "describe", "--tags", "--dirty"])
-                   .decode('ASCII').strip()
-                   .replace('-', '+', 1).replace('-', '.'))
-    with open(VERSIONFILE, "wt") as o:
-        o.write("__version__ = {!r}\n".format(__version__))
-
-except (subprocess.CalledProcessError, OSError):
-    if os.path.exists(VERSIONFILE):
-        __version__ = runpy.run_path(VERSIONFILE)['__version__']
-    else:
-        __version__ = "unknown"
+if os.path.exists(VERSIONFILE):
+    __version__ = runpy.run_path(VERSIONFILE)['__version__']
+else:
+    __version__ = "unknown"
